web/src/components/tasks/business-formation/review/ReviewForeignCertificate.tsx

```diff
@@ -0,0 +1,35 @@
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { InputFile } from "@businessnjgovnavigator/shared/formationData";
+import { ReactElement } from "rehype-react/lib";
+import { ReviewSectionHeader } from "./ReviewSectionHeader";
+
+interface Props {
+  foreignGoodStandingFile: InputFile | undefined;
+}
+
+export const ReviewForeignCertificate = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+  return (
+    <>
+      <ReviewSectionHeader
+        header="Foreign Certificate of Good Standing"
+        stepName="Business"
+        testId="foreign-certificate"
+      />
+      {props.foreignGoodStandingFile?.filename ? (
+        <div style={{ display: "flex", alignItems: "center" }}>
+          <img
+            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
```

Comment:
what is this?

Reply:
This is the preview image that's used by the USWDS file-input component. I used it here for continuity. I could extrapolate this to a global variable or reference it some other way if you'd like.

Reply:
lets give it a variable name so other people can know this also